schema/schema.graphql

```diff
@@ -3768,6 +3770,15 @@ type GitCommit implements Node {
     """
     externalURLs: [ExternalLink!]!
     """
+    The Git tree or blob in this commit at the given path.
+    """
+    path(
+        """
+        The path of the tree or blob.
+        """
+        path: String = ""
+    ): GitTreeOrBlob
+    """
```

Comment:
[Warning: GraphQL noob question] I'm a little confused... There is already a `GitTree` below and a `GitBlob` as well. Could you clarify why this field is needed?

Reply:
(this is an unrelated change that got synced)

`GitTreeOrBlob` appears to be an alias for `GitTree | GitBlob`.